### PR TITLE
fix: Link takes to the Github page even if click out side the button

### LINKED ISSUE #30: Link takes to the Github page even if click out side the button
If we click outside of the Star on GitHub button, it still takes the user to the GitHub page

image

What this issue propose is that only when clicking on the button we should take the user to the GitHub page.

### PR DESCRIPTION
**Pull request by Hat52**
Closes #30  
https://www.loom.com/share/7e31f2d3ed6449deb30513271b547969

### CODE REVIEW

**Comment by Sachin-chaurasiya** [comment]
Hey @Hat52 , thanks for the PR, there is a merge conflict in your please resolve it.

File: src/pages/HomePage/HomePage.tsx [hunk 1]
```diff
@@ -21,10 +21,12 @@ import { motion } from 'framer-motion';
 const HomePage = () => {
   return (
     <Grid gap={8}>
+	<Flex justifyContent="center">	
       <Link
         _hover={{ textDecoration: 'none' }}
         href="https://github.com/OpenSource-Journey/Your-Github-Contributions"
         target="_blank"
+		style={{width:'max-content'}}
```
Comment (Sachin-chaurasiya) [inline src/pages/HomePage/HomePage.tsx:29]: @Hat52 , Chakra UI supports CSS in JSX, so you don't have to use the style object to give the styling to the element.

For more you can refer this https://chakra-ui.com/getting-started
```suggestion
		width:'max-content'
                 margin='auto'
```

File: src/pages/HomePage/HomePage.tsx [hunk 2]
```diff
@@ -21,10 +21,12 @@ import { motion } from 'framer-motion';
 const HomePage = () => {
   return (
     <Grid gap={8}>
+	<Flex justifyContent="center">
```
Comment (Sachin-chaurasiya) [inline src/pages/HomePage/HomePage.tsx:24]: We don't need flex, `margin: auto` will work here.

**Review by Sachin-chaurasiya — CHANGES_REQUESTED** [review]
Left a few comments, thanks

**Comment by Sachin-chaurasiya** [comment]
@Hat52 , I think you have mixed up the changes of issues #27 and #30.

This PR should only include the changes for issue #30

**Comment by Hat52** [comment]
> @Hat52 , I think you have mixed up the changes of issues #27 and #30.
> 
> This PR should only include the changes for issue #30 

I have already created the PR for #27. the extra changes are of that issue. If you review that PR and merge it then we are good to go.

**Comment by Sachin-chaurasiya** [comment]
> > @Hat52 , I think you have mixed up the changes of issues #27 and #30.
> 
> > 
> 
> > This PR should only include the changes for issue #30 
> 
> 
> 
> I have already created the PR for #27. the extra changes are of that issue. If you review that PR and merge it then we are good to go.

I saw your another PR, my point was this PR should only have the changes for issue #30 only.

**Comment by Hat52** [comment]
> our

Yes, I got your point. The issue is I mistakenly made changes of the first PR in the main branch and after changes I switched to new branch.

Now I don't know how I can revert the changes from main branch

**Comment by Sachin-chaurasiya** [comment]
> > our
> 
> Yes, I got your point. The issue is I mistakenly made changes of the first PR in the main branch and after changes I switched to new branch.
> 
> Now I don't know how I can revert the changes from main branch

Hello @Hat52 , Here are the steps that you can follow.

- Switch to this PR branch `git checkout link-redirect`
- Remove the Image related changes and only keep the `Star on GitHub` button changes.
- Push the change to the `link-redirect` branch.

**Comment by Hat52** [comment]
@Sachin-chaurasiya Done

> > > our
> > 
> > 
> > Yes, I got your point. The issue is I mistakenly made changes of the first PR in the main branch and after changes I switched to new branch.
> > Now I don't know how I can revert the changes from main branch
> 
> Hello @Hat52 , Here are the steps that you can follow.
> 
> * Switch to this PR branch `git checkout link-redirect`
> * Remove the Image related changes and only keep the `Star on GitHub` button changes.
> * Push the change to the `link-redirect` branch.

@Sachin-chaurasiya Done

**Review by Sachin-chaurasiya — APPROVED** [review]
Great work @Hat52  🙌🏻,
Thanks for the contributions ✨

**Pull request merged**